samples/openthread/ncp/CMakeLists.txt

```
@@ -5,13 +5,10 @@
 #
 cmake_minimum_required(VERSION 3.13.1)
 
-set(NRF_SUPPORTED_BOARDS
-	nrf52840dk_nrf52840
-	nrf52833dk_nrf52833
-)
-
 set(OT_NCP_VENDOR_HOOK_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+list(APPEND OVERLAY_CONFIG ../common/overlay-ot-defaults.conf)
```

Using `${ZEPHYR_NRF_MODULE_DIR}` will allow downstream users of NCS to copy this sample to another location and use it as base for their own sample, and still have the correct path to the overlay file.

```suggestion
list(APPEND OVERLAY_CONFIG ${ZEPHYR_NRF_MODULE_DIR}/samples/openthread/common/overlay-ot-defaults.conf) 
```